arb_os/accounts.mini

```diff
@@ -362,6 +362,28 @@ public func account_setBaseTxFeeL1Gas(account: Account, fee: uint) -> Account {
     };
 }
 
+func entrypoint_stub() {}
+
+public func account_setStorage(account: Account, storage: StorageMap) -> Account {
+    return account with {
+        contractInfo: Some(
+            xif let Some(contractInfo) = account.contractInfo {
+                contractInfo with {
+                    storage: storage
+                }
+            } else {
+                struct {
```

Comment:
In the `else` case, it seems better to return a failure rather than creating an invalid contract. We might make `account_setStorage` return `option<Account>`, and then return `None` in the `else` case here.

Reply:
I improved the else case to behave like the revert precompile, and also treated "store" for a new contract.
We could get a long list of overrides, with duplicate entries etc.. it's possible that code override will follow storage override for a new contract, and possible that we'll get a storage override that will eventually be irrelevant for the call. I think it's better not to fail unless we need to.
If call ends up calling contract that has no code - it should fail. I think revert_precompile behavior makes sense..